promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,38 @@
+# Connect Promscale and Jaeger Query
+This section shows you how to connect Promscale with
+[Jaeger][grafana-jaeger-datasource].
+
+Before you begin, you need the IP address of your Promscale instance. You can
+find this at the command prompt, using this command:
+```bash
+docker inspect <PROMSCALE_CONTAINER_NAME>
+```
+
+The IP address is listed in the `NetworkSettings → Networks → IPAddress`
+section.
+
+Alternatively, you can set the URL as `promscale:9202` (here `9202` is the gRPC endpoint of Promscale), where `promscale` is the name of the container.
+
+<procedure>
+
+### Connecting Promscale and Jaeger Query
+1.  Install Jaeger from the [official Docker image][jaeger-docker]:
```

Comment:
You don't need to install Jaeger if you already have it. We should explain what are the configuration parameters you need to pass to connect it to Promscale. Also indicate what's the minimum Jaeger version that supports that.